scripts/generator.py

```diff
@@ -41,7 +41,7 @@ def main():
     # ecs_helpers.yaml_dump('ecs.yml', fields)
 
     fields = loader.load_schemas(ref=args.ref, included_files=args.include)
-    cleaner.clean(fields)
+    cleaner.clean(fields, strict_mode_enabled=args.strict)
```

Comment:
Can we call this `strict` everywhere? Let's keep it terse until there's a reason to have more complex names :-)